migrations/V180__System_Intake_Contract_Number_Migration.sql

```diff
@@ -0,0 +1,37 @@
+INSERT INTO system_intake_contract_numbers
+(
+    id,
+    system_intake_id,
+    contract_number,
+    created_by,
+    created_at,
+    modified_by,
+    modified_at
+)
+SELECT
+    gen_random_uuid(),
+    si.id,
+    si.contract_number,
+    (SELECT
+        CASE
+            WHEN
+                si.eua_user_id IS NULL
+            THEN
+                '00000001-0001-0001-0001-000000000001' -- EASI SYSTEM USER
+            ELSE
+                (SELECT ua.id FROM user_account ua WHERE ua.username = si.eua_user_id)
+	    END),
```

Comment:
Per our call, I think we should just replace this with a hard-coded system user, due to the fact that not every EUA that has an associated intake will currently have a user account.

```suggestion
    '00000001-0001-0001-0001-000000000001' -- EASI SYSTEM USER

```

Reply:
Ok makes sense! So we can attribute all creation/modified to the EASI SYSTEM USER and then record live data going forward

Reply:
Yeah, I think that's the right path to take on this, since we're not really utilizing the `created` and `modified` metadata much currently, I don't want to sink too much effort into it!

Reply:
Awesome! Thanks Clay 😄